model-server-openapi/specifications/model-server.yaml

```diff
@@ -13,481 +13,700 @@ info:
   description: "modelix public API"
   version: "1.0.0"
 servers:
-  - url: '/'
+  - url: '/v2'
```

Comment:
Do we want version throug an URL?
How to we evolve the API?
Do we want to use MediaType-Versioning?
https://opensource.zalando.com/restful-api-guidelines/#114